### PR TITLE
A formatter for p-values

### PR DESCRIPTION
**Pull request by larmarange**
cf. https://github.com/r-lib/scales/issues/144

### CODE REVIEW

**Comment by larmarange** [comment]
Just a quick question: is it better to have an `accuracy` argument, consistent with `number_format` or would you prefer for that specific formatter a `digits` argument?

File: R/formatter.r [hunk 1]
```diff
@@ -438,3 +438,56 @@ time_format <- function(format = "%H:%M:%S", tz = "UTC") {
...
+#' @export
+#' @examples
+#' p <- c(.50, 0.12, .045, .011, .009, .00002, NA)
+#' pvalue(p)
+#' pvalue(p, accuracy = .01)
+#' pvalue(p, add_p = TRUE)
+#' custom_function <- pvalue_format(accuracy = .1, decimal.mark = ",")
+#' custom_function(p)
+pvalue_format <- function(accuracy = .001, decimal.mark = ".", add_p = FALSE) {
+  function(x) pvalue(
```
Comment (hadley) [inline R/formatter.r]: Need to force the 3 arguments

**Comment by larmarange** [comment]
done

**Comment by hadley** [comment]
Thanks!

**Pull request merged**